### ISSUE TITLE
Update style of  Operating Status alert on VAMC Location pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11700]
## Description
Within the VAMC locations pages, when a location has an operating status other than normal, an alert displays. This alert was created prior to the VA Design System and therefore does not exactly match the current DS standards. For accessibility as well as UX/design integrity, it would be beneficial to update the styles of this element to be consistent with the VA design components.

### Design System Resources

- [Alert](https://design.va.gov/components/alert)
- [Alert - Expandable](https://design.va.gov/components/alert-expandable)
- [Link](https://design.va.gov/components/link/)

## Suggested updates

- The design system indicates that a link should be underlined.  As these operating statuses are links to another page, it should be underlined to follow the expected pattern and indicate to users that it is a link
- Alerts all use a bolder font. Again, to follow the expected pattern that has been set, these operating status alerts should be updated to have the bolder font that the DS has now set.
- Hover state
  - The current hover states of the alerts is a slight darkened background color and not very visible
  - The alert components in contrast have a hover state that is much more prominent and a more visible change
 
### Screenshot of existing states:
- Facility notice blue state:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/3a6c8389-69e6-4703-81a4-8ee2ef727f78)

- Limited hours warning yellow state:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/b9c15edf-7828-44e8-906d-1d848ec0afc6)

### Example of what the warning operating status would look like that follows these suggested updates:

**Normal State:**
![image](https://user-images.githubusercontent.com/106678594/203622810-1cd86e58-633c-4954-8e85-32a1f816c670.png)

**Hover State:**
![image](https://user-images.githubusercontent.com/106678594/203623125-85d8ea7a-5ae1-42f8-bfdb-abb91d3247a4.png)

## Acceptance Criteria
- [ ] Underline Operating Status Alert links
- [ ] Make Operating Status Alert link text bold
- [ ] Make Operating Status Alert match DS expandable alert hover states
   - [ ] "Facility Notice" should match the blue Default alert
   - [ ] "Limited Services and hours" should match the yellow Warning alert
   - [ ] "Facility Closed" should match the red Error alert
- [ ] UX/Design Review with @laflannery and @thejordanwood 

## Timing/Priority
While this is an accessibility issue, it is not a failure of 508 and therefore does not need to be considered a high priority.  There is a possible consideration for completing this within the current COVID status updates or while completing the work for #11699 as it may be more efficient.  If that is not the case and this will be completed separately, a suggested timeline would be perhaps the start of Q1.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@davidmpickett If you would like to change/edit anything in the description above please feel free as these are mostly UX/Design updates and my main concern is that these alerts are consistent across the site.

**Comment by davidmpickett** [comment]
This all looks good to me.

**Comment by dsinla** [comment]
@laflannery can we get a little more clarity on the existing state vs the desired state, and the exact components.
Can we also get more clarity on the Acceptance Criteria  - to more fully annotate what's changing? 
@dsinla will create a ticket for deeper DSN request.

**Comment by laflannery** [comment]
@dsinla Updated ACs

**Comment by mmiddaugh** [comment]
@kmariepat-cityfriends we may want to look at the implementations across the products. Because of the evolution in design and the roll-out timeline, it is likely these changes will also need to be reflected in displays in Facility Locator search results and on Vet Center main and location pages. 

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/40f284af-8f52-41cc-80df-a6f84cb84ea7)

**Comment by laflannery** [comment]
@mmiddaugh This exact implementation (An alert that only links out to another page) is not actually a component in the DS - which is why it doesn't quite follow the guidelines set forth in other components. The VAMC Location pages I believe is the only instance of this particular usage of this linking alert.

The other instances you mentioned are in the Design system, and while it does look like our styles are accurate, the update we may want to consider for consistency and to future proof this for any update the DS team may make would be to create separate tickets to update the existing alerts to use the DS components.  For example on [Vet Center pages](https://www.va.gov/springfield-ma-vet-center/) the style of the expandable alert matches what is in the [VA guide](https://design.va.gov/storybook/?path=/docs/components-va-alert-expandable--default) but the HTML does not have `<va-alert-expandable>
![image](https://user-images.githubusercontent.com/106678594/236909552-ccde9111-34df-4b72-8572-7eae980393aa.png)

**Comment by kmariepat-cityfriends** [comment]
Next Steps:
- Determine the LOE between fixing accessibility issues noted in AC AND making a new design component 

@maxx1128 can you comment on the LOE difference in order to determine which approach we should take

**Comment by kmariepat-cityfriends** [comment]
Look into design experimental pattern submission --> includes some metrics to justify

**Comment by maxx1128** [comment]
@kmariepat-cityfriends I think the LOE would be higher for making the new design component, since there's a much longer and often slower process in getting it right and set up. I would personally go with directly fixing the accessibility issues first and make the new design component a longer-term fix to gradually replace the short-term one.

**Comment by jilladams** [comment]
From refinement: 
* We are not currently using a DS component. 
* We _are_ using an amalgamation / Franken-build that references other DS components
* Laura's goal: align our Franken-build with existing DS component treatments, so be more in line with DS.
* Screenshots in ticket body show regular state first, then hover state (hover hand included)
    * This yellow treatment is, we think, a warning state
    * The ACs articulate that we need to build the remaining states, we just don't have screen shots of those

**Comment by laflannery** [comment]
I added a section for current status screenshots, which includes a blue facility notice example I was able to found now

**Comment by mmiddaugh** [comment]
@laflannery I found examples for "Facility closed"  if you want to comment on these (as per usual they are different for VAMC than Vet Centers in current state

<details>
<summary> Facility closed - Helena VA Clinic </summary> 

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/a50e9c5d-a70f-4209-b5ac-65fb1d9fe6ba)
</details> 

<details>
<summary> Facility closed - Mesquite Vet Center CAP example </summary> 

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/6c9e9f62-ddbc-4927-ad5f-677d160e67c2)
</details>

**Comment by jilladams** [comment]
@maxx1128 PR merged -- is this verified in Prod or ready to be verified in Prod?

**Comment by maxx1128** [comment]
@jilladams Ready to be verified in Prod.

**Comment by jilladams** [comment]
note: @laflannery flagged that a11y review is outstanding here. This is partially my bad, as well, for closing and not double checking ACs. @maxx1128 In the absence of Kamari, I need your help to flag when ACs need other eyes or if you need support. For now, I'm not going to reopen the ticket, but please make sure to get Laura's & Jordan's feedback to confirm the fix, and reopen ticket if there are any notes / changes required.

@thejordanwood @laflannery please take a look when you have an opportunity and let us know!

**Comment by laflannery** [comment]
This looks good to me. @thejordanwood I wasn't able to find any current Facility Closed or Limited Services notices but there are a couple examples of the default blue (hopefully they will still be active when you take a look!)
- [Facility notice at Yonkers](https://www.va.gov/bronx-health-care/locations/yonkers-va-clinic/)
- Another Facility notice if the one above is gone: [Indianapolis YMCA](https://www.va.gov/indiana-health-care/locations/indianapolis-ymca-va-clinic/)

**Comment by thejordanwood** [comment]
@laflannery Thanks! This looks good to me too.
